decidim-meetings/spec/system/user_edit_meeting_spec.rb

```diff
@@ -28,6 +28,33 @@
     stub_geocoding(meeting.address, [latitude, longitude])
   end
 
+  describe "editing meeting with empty description" do
+    before do
+      login_as user, scope: :user
+      organization.update(rich_text_editor_in_public_views: true)
+    end
+
+    it "when WYSIWYG is active" do
```

Comment:
Here you actually set that the content to "<p>[space]</p>". 

I would test 2 things in the same spec, to make sure everything is ok. First set the meeting form to be valid, then edit it, and set the description to empty to see if is invalid.